portfolio/src/main/webapp/index.html

```diff
@@ -10,14 +10,60 @@
       <!-- After added content and css styling report back to step 11 -->
       <!-- to access the walkthrough: teachme ~/software-product-sprint/walkthroughs/week-1-web-development/portfolio-walkthrough.md -->
     <div id="content">
-      <img class="main-image" src="images/LinkedIn.png"/>
+      <img class="image" src="images/LinkedIn.png"/>
       <h1>Samantha's Portfolio</h1>
       <p>This is my portfolio.</p>
       <p>Hello! My name is Samantha Murray Tuesta and I am a second-year undergraduate student at Western Washington University with a passion for all things computer science! Welcome to my portfolio!</p>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to learn a fun fact about me:</p>
       <button onclick="addFunFact()">Fact</button>
       <div id="greeting-container"></div>
+      <img class="profile-pic" src="images/profilePicture.JPG"/>      
+      <h3>About Me</h3>
+      <p>I am a first generation college student, in my second year studying Computer Science and Internet Studies. I am currently based in Seattle, WA, and attend college in Bellingham!
+         <br><br>At my institution, Western Washington University, I am engaged in research specializing in digital privacy and its portrayal in the media. My work is aimed at creating a digital web game to engage students of all ages and teach the importance of digital privacy and how we go about advocating for those impacted by breaches in that privacy.
```

Comment:
You are using `<br>` elements to control the visual appearance? I think that this runs contrary to the aspirational goal of HTML/CSS in that tags should express document semantic structure while CSS should control how it's rendered to the page. Consider using a `<p>` element for each block and having a CSS class that controls spacing?

Reply:
The HTML has now been changed to <p> elements with CSS styling to adjust its proximity to other elements on the page!